template/src/services/asyncStorage/storage.js

```diff
@@ -0,0 +1,29 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+export const clearAllData = async () => {
+  try {
+    await AsyncStorage.clear()
+  } catch (e) {}
+}
+
+export const setData = async (key, data) => {
```

Comment:
keep the func name is `setString` so we don't need to change too much

Reply:
I think we need use setData name which have more meaningful